gallery/src/components/NxStatefulGlobalSidebar/NxStatefulGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,204 @@
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>toggleOpenIcon</NxTable.Cell>
+                <NxTable.Cell>IconDefinition</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  An icon used by the sidebar's open/close toggle when the sidebar is in the open state. The icon
+                  should make it clear that clicking it will close the sidebar.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>toggleClosedIcon</NxTable.Cell>
+                <NxTable.Cell>IconDefinition</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  An icon used by the sidebar's open/close toggle when the sidebar is in the closed state. This icon
+                  should make it clear that clicking it will open the sidebar.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>logoImg</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  The path to an image placed in the top left corner which displays branding and product name.
+                  <strong>Note:</strong> the image should not be larger than 145px wide and 32px tall. It should
+                  display either a logo (apx 32px x 32px) and the product name, or just the product name.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>logoAltText</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>Alt text for the logo</NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>logoLink</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  When the logo is clicked it navigates to a page (typically Home) specified here.
+                </NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Body>
+          </NxTable>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <h3 className="nx-h3"><NxCode>NxStatefulGlobalSidebarNavigation</NxCode> Props</h3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            <NxCode>NxStatefulGlobalSidebarNavigation</NxCode> is a container for navigation links.
+            It accepts all standard <NxCode>&lt;div&gt;</NxCode> HTML attributes.
+          </NxP>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <h3 className="nx-h3"><NxCode>NxStatefulGlobalSidebarNavigationLink</NxCode> Props</h3>
```

Comment:
These aren't things

Reply:
> These aren't things

Got a little exuberant with my search and replace: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/3a85d6147904a438c5742677e401a7c9797ec515

Reply:
> Got a little exuberant with my search and replace: 3a85d61

FWIW I'm not sure duplicating all of the documentation is a good idea, as it means we'll have to remember to make any changes in both places.  I'd shorten the stateful page docs to defer to the stateless ones for most things.

Reply:
> > Got a little exuberant with my search and replace: [3a85d61](https://github.com/sonatype/sonatype-react-shared-components/commit/3a85d6147904a438c5742677e401a7c9797ec515)
> 
> FWIW I'm not sure duplicating all of the documentation is a good idea, as it means we'll have to remember to make any changes in both places. I'd shorten the stateful page docs to defer to the stateless ones for most things.

Ah, yes good point: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/6cd6ba526d753b12f7b0f2d21f9f0a8d2bb8a794

Reply:
> Ah, yes good point: 6cd6ba5

Might want to make clear that it has `defaultOpen` _instead of_ `isOpen`

Reply:
> > Ah, yes good point: [6cd6ba5](https://github.com/sonatype/sonatype-react-shared-components/commit/6cd6ba526d753b12f7b0f2d21f9f0a8d2bb8a794)
> 
> Might want to make clear that it has `defaultOpen` _instead of_ `isOpen`

https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/d45e57ab9e169b3c9d39e5c832364c74aca4bc25